### PR TITLE
Login Interface (Frontend part)

### PR DESCRIPTION
**Pull request by lauralindal**
solves #41 as well as #44 
- Header now has a Login Component
- Cleaned up rendering in index.html
- included simple Login interface (no functionality yet)

Please review @noobiba @schultyy

### CODE REVIEW

File: src/Login.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        <div className="three columns">
+          <label htmlFor="Username">Username</label>
+          <input type="text" placeholder="Username" id="Username" />
+        </div>
+        <div className="three columns">
+          <label htmlFor="Username">Password</label>
+          <input type="password" placeholder="Password" id="Password" />
+        </div>
+        <div className="two">
+          <label htmlFor="exampleEmailInput">&nbsp;</label>
```
Comment (noobiba) [inline src/Login.js]: There is an email field which is actually not visible.

**Review by noobiba — COMMENTED** [review]
@lauralindal Please take a look at "<label htmlFor="exampleEmailInput">&nbsp;</label>". Seems that this will not be displayed. Is this your intention?

**Review by noobiba — COMMENTED** [review]
@lauralindal Please take a look at my comment in src/Login.js. thank you.

File: src/Login.js (hunk 1 above)
Comment (schultyy) [reply src/Login.js]: I think the email field is completely missing, no?

**Review by schultyy — CHANGES_REQUESTED** [review]
As Tanja already commented, the email field is missing.

**Review by schultyy — APPROVED** [review]
lgtm

File: src/Login.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+class Login extends React.Component {
+  render() {
+    return (
+      <form>
+        <div className="three columns">
+          <label htmlFor="Email">Email Address</label>
+          <input type="email" placeholder="Email Address" />
+        </div>
+        <div className="three columns">
+          <label htmlFor="Username">Password</label>
```
Comment (noobiba) [inline src/Login.js:12]:
```
<label htmlFor="Username">Password</label>
```
should be
```
 <label htmlFor="Password">Password</label>
```

File: src/Login.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+class Login extends React.Component {
+  render() {
+    return (
+      <form>
+        <div className="three columns">
+          <label htmlFor="Email">Email Address</label>
+          <input type="email" placeholder="Email Address" />
+        </div>
+        <div className="three columns">
```
Comment (noobiba) [inline src/Login.js:11]: Where is the username field gone..?
 ```
<label htmlFor="Username">User name</label>       
<input type="username" placeholder="Username" />
```

**Comment by noobiba** [comment]
I left single line comments. The username field is missing. The order of the fields might be reviewed. Name of placeholder is wrong.
@lauralindal @schultyy  Please review again. Thank you. :-)

**Comment by lauralindal** [comment]
Sorry @noobiba this has already been merged. Will fix in another ticket. However, we don't need username *and* email - we will just use email :)

**Pull request merged**